.github/actions/vmtest/build_selftests.sh

```diff
@@ -6,9 +6,19 @@ THISDIR="$(cd $(dirname $0) && pwd)"
 
 source "${THISDIR}"/helpers.sh
 
-travis_fold start prepare_selftests "Building selftests"
+TOOLCHAIN="$1"
+TOOLCHAIN_NAME="$(echo $TOOLCHAIN | cut -d '-' -f 1)"
+TOOLCHAIN_VERSION="$(echo $TOOLCHAIN | cut -d '-' -f 2)"
 
+if [ "$TOOLCHAIN_NAME" == "llvm" ]; then
+export LLVM="-$TOOLCHAIN_VERSION"
```

Comment:
isn't it always LLVM=1 ?

Reply:
btw, @yonghong-song, FYI, we'll soon start detecting all these LLVM-specific issues early on. Please take a look at this as well.

Reply:
LLVM toolchain we install in CI has version suffix (currently it is 15).

Linux Kernel build system handles this gracefully: https://github.com/torvalds/linux/blob/b4a5ea09b29371c2e6a10783faa3593428404343/tools/scripts/Makefile.include#L54-L73

I do not like however that it still defaults to clang without version suffix: https://github.com/torvalds/linux/blob/b4a5ea09b29371c2e6a10783faa3593428404343/tools/scripts/Makefile.include#L87-L92. This functionality is used when we build bpf c code in selftests (https://github.com/kernel-patches/vmtest/pull/82/files#diff-c34c251202780f09ab91ab0021a7190227e9a48d8063b004a644fe65a365125fR37-R39).

I am thinking about sending a patch to Linux build system to properly initialize CLANG variable when LLVM variable is set and has a version suffix provided

Reply:
Some more details in https://www.kernel.org/doc/html/latest/kbuild/llvm.html

Reply:
nice, didn't know about it.

Reply:
Looks like this llvm suffix thing is just extended two months ago:
  https://lore.kernel.org/all/20200317215515.226917-1-ndesaulniers@google.com/
the llvm prefix thing is extended earlier.

I don't have a strong opinion on this as I always use the default path 'clang' only. Whatever it is best for CI is also okay for me.